src/Prismic/Dom/RichText.php

```diff
@@ -32,7 +32,7 @@ public static function asText($richText)
             }
         }
 
-        return $result;
+        return substr($result, 0, -1);
```

Comment:
How do you know for sure the last character is a new line? What if a future change means that \n is no longer present. Why is it even important to trim newlines? Anyhow, I would personally use `trim` here as at least you know it will only remove whitespace.

Reply:
I totally agree with you. I should have used `trim()` and will change it tomorrow